test/core/modules1/canvas-resize.js

```diff
@@ -0,0 +1,34 @@
+var test = require('tape');
+
+require('../../../src/ImageSequencer.js');
+
+var sequencer = ImageSequencer({ ui: false });
+var options = { width: 500, height: 500 };
+var red = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABlBMVEX+AAD///+KQee0AAAAAWJLR0QB/wIt3gAAAAd0SU1FB+EGHRIVAvrm6EMAAAAMSURBVAjXY2AgDQAAADAAAceqhY4AAAAldEVYdGRhdGU6Y3JlYXRlADIwMTctMDYtMjlUMTg6MjE6MDIrMDI6MDDGD83DAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE3LTA2LTI5VDE4OjIxOjAyKzAyOjAwt1J1fwAAAABJRU5ErkJggg==';
+sequencer.loadImages(red);
+sequencer.addSteps('canvas-resize', options);
+
+// // Test 1 to check brightness module is getting loaded
```

Comment:
Are we going to use this commented code soon?

Reply:
Removing it!! Left by mistake.
Thanks for pointing out!!